lib/compat/wordpress-6.5/fonts/class-wp-font-collection.php

```diff
@@ -48,28 +48,32 @@ final class WP_Font_Collection {
 		 *
 		 * @since 6.5.0
 		 *
-		 * @param string       $slug         Font collection slug.
-		 * @param array|string $data_or_file Font collection data array or a path/URL to a JSON file
-		 *                                   containing the font collection.
-		 *                                   See {@see wp_register_font_collection()} for the supported fields.
+		 * @param string $slug Font collection slug. May only contain alphanumeric characters, dashes,
+		 *                     and underscores. See sanitize_title().
+		 * @param array  $args Font collection data. See wp_register_font_collection() for information on accepted arguments.
 		 */
-		public function __construct( $slug, $data_or_file ) {
+		public function __construct( string $slug, array $args ) {
 			$this->slug = sanitize_title( $slug );
 			if ( $this->slug !== $slug ) {
 				_doing_it_wrong(
 					__METHOD__,
 					/* translators: %s: Font collection slug. */
-					sprintf( __( 'Font collection slug "%s" is not valid. Slugs must use only alphanumeric characters, dashes, and underscores.', 'gutenberg' ), $slug ),
+					sprintf( __( 'Font collection slug "%s" is not valid. Slugs must use only alphanumeric characters, dashes, and underscores.' ), $slug ),
```

Comment:
Wondering if we need to add `'gutenberg'` back in this case?

Reply:
yes, thanks for pointing that out